pages/common/fortune.md

```diff
@@ -0,0 +1,27 @@
+# fortune
+
+> Print a random quotation (fortune-cookie style)
+
+- Print a quotation
+
+`fortune`
+
+- Print a quotation from a given database
+
+`fortune {{database}}`
+
+- Print a list of quotation databases
```

Comment:
How about "List the files where the quotations are sourced from"? I think using "files" is preferable because it matches with the short option name (`-f`)